CONTRIBUTING.md

```diff
@@ -135,8 +135,6 @@ After adding a new example, make sure to run `yarn build:example <examplename>`
 
 - `test/` - Code for unit testing. `test`'s structure reflects `src`'s directory structure.
 For example, `test/compile/` tests files inside `src/compile/`.
-  - Note that we prepend `/* tslint:disable:quotemark */` to all files under `test/compile`
```

Comment:
Why removing this?  Are you removing double quotes from all tests?  (I'd rather keep the double quote so we can copy spec to editor to test easily.)

Reply:
We don't use this consistently (aka when I wanted to use it, I coudln't) and now that the tests are much faster, I feel this is less necessary.

Reply:
Why is it related to faster test?

Reply:
http://bit.ly/js2json